### PR TITLE
Tim rack schema feature for database

### PR DESCRIPTION
**Pull request by ChrisWilhelm**

### CODE REVIEW

File: server/prisma/schema.prisma [hunk 1]
```diff
@@ -22,4 +22,31 @@ model Distributor {
   id Int @id @default(autoincrement())
   name String
   user User[] @relation("isPointOfContact")
+}
+
+
+model Food_Item {
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:28]: make this `FoodItem`

File: server/prisma/schema.prisma [hunk 2]
```diff
@@ -22,4 +22,31 @@ model Distributor {
...
+  id Int @id @default(autoincrement())
+  pallet_id String
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:30]: palletId

File: server/prisma/schema.prisma [hunk 3]
```diff
@@ -22,4 +22,31 @@ model Distributor {
...
+  common_name String
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:31]: just name

File: server/prisma/schema.prisma [hunk 4]
```diff
@@ -22,4 +22,31 @@ model Distributor {
...
+  pallet Pallet @relation(fields: [id], references: [id])
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:32]: you need to add a palletId field and then @relation(fields:[palletId], references: [id])

File: server/prisma/schema.prisma [hunk 5]
```diff
@@ -22,4 +22,31 @@ model Distributor {
...
+  weight String
+  description String
+}
+
+model Pallet {
+  id Int @id @default(autoincrement())
+  rack_id String
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:39]: rackId

File: server/prisma/schema.prisma [hunk 6]
```diff
@@ -22,4 +22,31 @@ model Distributor {
...
+  distributor_name String
+  rack Rack @relation(fields: [id], references: [id])
+  food_items Food_Item[]
+  weight String
+  description String
+}
+
+model Rack {
+  id Int @id @default(autoincrement())
+  location_number_and_letter String   // 1A, 2B, etc
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:49]: just location

File: server/prisma/schema.prisma [hunk 7]
```diff
@@ -22,4 +22,31 @@ model Distributor {
...
+  pallet Pallet @relation(fields: [id], references: [id])
+  weight String
+  description String
+}
+
+model Pallet {
+  id Int @id @default(autoincrement())
+  rack_id String
+  distributor_name String
+  rack Rack @relation(fields: [id], references: [id])
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:41]: again need a rackId, see above

File: server/prisma/schema.prisma [hunk 8]
```diff
@@ -22,4 +22,31 @@ model Distributor {
...
+  food_items Food_Item[]
```
Comment (ChrisWilhelm) [inline server/prisma/schema.prisma:42]: change to foodItems

**Comment by i-api** [comment]
I fixed them!

**Pull request merged**